v0/destinations/facebook_pixel/transform.js

```diff
@@ -527,6 +527,16 @@ const responseBuilderSimple = (message, category, destination) => {
     }
   }
 
+  // content_category should only be a string ref: https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/custom-data
+
+  if (
+    customData &&
+    customData.content_category &&
+    typeof customData.content_category !== "string"
+  ) {
+    delete customData.content_category;
```

Comment:
Why are we deleting it instead of throwing an error?

Reply:
We should throw error here rather than deleting

Reply:
resolving conversation as changes are approved.